composer.json

```diff
@@ -29,7 +29,9 @@
         "behat/web-api-extension": "~1.0@dev",
         "behat/symfony2-extension": "~2.0@dev",
         "phpbench/phpbench": "~0.10.0",
-        "phpbench/tabular": "@dev"
+        "phpbench/tabular": "@dev",
+
+        "guzzlehttp/streams": "^3.0.0"
```

Comment:
if i remove this line the `--prefer-lowest` test doesnt work anymore (because of composer) https://travis-ci.org/massiveart/MassiveSearchBundle/jobs/140464875#L182